src/test/scala/org/broadinstitute/dsde/firecloud/service/ElasticSearchDAOQuerySupportSpec.scala

```diff
@@ -373,7 +375,9 @@ class ElasticSearchDAOQuerySupportSpec extends FreeSpec with ElasticSearchDAOQue
             boolMap.fields("should") match {
               case shouldArray:JsArray =>
                 shouldArray
+              case x => throw new Exception(s"unmatched case for  ${x.getClass.getName}: ${x.toString()}")
             }
+          case x => throw new Exception(s"unmatched case for  ${x.getClass.getName}: ${x.toString()}")
```

Comment:
changes in this file are not in scope for this PR, but they fix compile warnings that were bugging me.